pennylane/math/quantum.py

```diff
@@ -164,6 +169,167 @@ def marginal_prob(prob, axis):
     return np.flatten(prob)
 
 
+def _density_matrix_from_matrix(density_matrix, wires, check_state=None):
+    """Compute the density matrix from a state vector.
+
+    Args:
+        density_matrix (tensor_like): 1D tensor state vector. This tensor should be of size ``(2**N,)`` for some
+            integer number of wires``N``.

```

Comment:
This should be a 2D tensor with shape (2\*\*N, 2\*\*N).

Reply:
Thank you!